README.md

```diff
@@ -1,7 +1,13 @@
 # Forta Agent Templates
```

Comment:
Change all mentions of `Agent` to `Bot`.

Reply:
It might make sense for MR legibility sake to do agent -> bot conversion in a separate MR. This is already a headache to track, switching agent->bot might break the rest of the barely working diffs